src/main/java/fi/fmi/avi/archiver/message/MessageModifier.java

```diff
@@ -3,6 +3,6 @@
 @FunctionalInterface
 public interface MessageModifier {
 
-    AviationMessage modify(AviationMessage message);
+    AviationMessage.Builder modify(AviationMessage.Builder message);
```

Comment:
When modifying a builder, I think there should be no need to return anything. Especially when the return value is not being used. This API is yet going to change anyway, so I'll accept this now.